tests/capture/test_capture_transforms.py

```diff
@@ -199,3 +202,255 @@ def g(i):
 
         assert qfunc_jaxpr.eqns[1].primitive == qml.Z._primitive
         assert qfunc_jaxpr.eqns[2].primitive == qml.measurements.ExpectationMP._obs_primitive
+
+
+# pylint: disable=protected-access, expression-not-assigned
+class TestMapWiresTransform:
+    """Unit tests for the MapWiresTransform class."""
+
+    def test_qnode_simple(self):
+        """Test that a simple qnode is transformed correctly."""
+
+        const = 0.1
+
+        @MapWires(wire_map={0: 1, 1: 2, 2: 3, 3: 2})
+        @qml.qnode(qml.device("default.qubit", wires=4))
```

Comment:
For the purposes of testing, I think it makes sense to test without qnodes, and have one test to verify that transforming a qnode works correctly.

Reply:
Here I explicitly tested the workflow with qnodes because this was the main requirement of the story (although we currently don't have a good way to check the `jaxpr` without using a lot of assertions).

What do you think about one test with and one test without `qnode` for each (that is, one for `cond`, one for `for_loop`, etc.)?

Reply:
Generally the user stories are going to be in terms of a user workflow, but we want most tests to be as localized as possible.  When testing, we want to fail fast, and to have the failure point directly to the source of the problem. And "something went wrong in an integrated workflow" is a lot harder to interpret a very localized, targeted test.